### ISSUE TITLE
Error during recording the stream

### ISSUE THREAD

**Issue by howgermcgredy** [HoloArchivists/fc2-live-dl#40]
![image](https://user-images.githubusercontent.com/116721444/230152685-37b32990-5577-4d44-bc06-17967e510e81.png)
I had "typeerror: object of type 'int' has no len()" when someone is live streaming.
I wonder why it appear cause I didn't have this question before.
The picture is the code I have, thanks for the help first.

**Comment by hizkifw** [comment]
There was a change on FC2's side. This has been fixed in the latest version ([v2.1.2](https://github.com/HoloArchivists/fc2-live-dl/releases/tag/v2.1.2)). Please update by following the install guide in the README.

Using pip:

```
pip install --upgrade fc2-live-dl
```

Using docker:

```
docker pull ghcr.io/holoarchivists/fc2-live-dl:latest
```

**Comment by howgermcgredy** [comment]
> There was a change on FC2's side. This has been fixed in the latest version ([v2.1.2](https://github.com/HoloArchivists/fc2-live-dl/releases/tag/v2.1.2)). Please update by following the install guide in the README.
> 
> Using pip:
> 
> ```
> pip install --upgrade fc2-live-dl
> ```
> 
> Using docker:
> 
> ```
> docker pull ghcr.io/holoarchivists/fc2-live-dl:latest
> ```

It's working when I upgrate it.Thanks a lot.

**Comment by IAmVisco** [comment]
Weirdly enough, getting this on 2.2.0 (updated to it yesterday). I thought it was running old code so I rebooted the server a couple times but it's still spamming that during the download (the download and the file are fine though).
